app/src/main/java/com/glia/exampleapp/GliaWidgetsConfigManager.kt

```diff
@@ -74,29 +78,8 @@ object GliaWidgetsConfigManager {
             .apply()
     }
 
-    private fun obtainConfigFromSecretDeepLink(
-        data: Uri,
-        applicationContext: Context
-    ): GliaWidgetsConfig {
-        return GliaWidgetsConfig.Builder()
-            .setSiteApiKey(
-                SiteApiKey(
-                    data.getQueryParameter(API_KEY_ID_KEY)!!, data.getQueryParameter(
-                        API_KEY_SECRET_KEY
-                    )!!
-                )
-            )
-            .setSiteId(data.getQueryParameter(SITE_ID_KEY))
-            .setRegion(REGION_ACCEPTANCE)
-            .setContext(applicationContext)
-            .build()
-    }
-
     @JvmOverloads
-    fun createDefaultConfig(
-        applicationContext: Context,
-        uiJsonRemoteConfig: String? = null
-    ): GliaWidgetsConfig {
+    fun createDefaultConfig(applicationContext: Context, uiJsonRemoteConfig: String? = null): GliaWidgetsConfig {
```

Comment:
`@JvmStatic` annotation will help to use this method directly/without `INSTANCE` in the java code

Reply:
Thanks. That much nicer then INSTANCE option 👍

Reply:
Fixed in bdb1e06